operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -1354,6 +1354,8 @@ message TelemetryV2StackDriverConfig {
 
   google.protobuf.BoolValue topology = 4;
 
+  google.protobuf.BoolValue disableOutbound = 6;
```

Comment:
are we just going to have some request a `disableInbound` flag a couple of weeks from now, or do we think this is a good long term API?

Reply:
Disable client side reporting should be a general enough option that will last for long term.